docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-242.adoc

```diff
@@ -0,0 +1,71 @@
+
+== Azure Synapse Spark Pool is not using isolated compute
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/AzureSparkPoolIsolatedComputeEnabled.py[CKV_AZURE_242]
+
+|Severity
+|INFO
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan,ARM,Bicep
+
+|===
+
+=== Description
+
+This policy ensures that isolated compute is enabled for Synapse Spark pools in Azure. Isolated compute involves using dedicated resources that are not shared with other customers. This enhances security by minimizing the risk of data breaches or unauthorized access, as the compute resources are isolated to your environment. Enabling isolated compute is crucial for maintaining high security standards, especially when handling sensitive data or running critical applications in Synapse Spark pools.
```

Comment:
This policy detects whether isolated compute is disabled for Synapse Spark pools in Azure. Isolated compute utilizes dedicated resources that are not shared with other customers. This isolation minimizes data breach and unauthorized access risks, enhancing security for your Spark pools. Enabling isolated compute is crucial for maintaining high security standards, especially when handling sensitive data or running critical applications within Synapse Spark pools.